invenio_oauthclient/contrib/cern.py

```diff
@@ -212,13 +212,34 @@ def account_setup(remote, token):
 
     if response.status == requests.codes.ok:
         res = get_dict_from_response(response)
-        current_user.info['group'] = fetch_groups(res['Group'])
+        groups = fetch_groups(res['Group'])
+        current_user.info['group'] = groups
         current_user.modified = True
         current_user.save()
 
-        if user and not any([user.family_name, user.given_names]):
-            user.family_name = res['Lastname'][0]
-            user.given_names = res['Firstname'][0]
-
-            db.session.add(user)
-            current_user.reload()
+        # update user object
+        if user:
```

Comment:
is it possible that we don't get a user from https://github.com/inveniosoftware/invenio-oauthclient/pull/7/files#diff-b49dd27458efb60f2de74978e6d3eecfR211?